### PR TITLE
Webapp  example

### PR DESCRIPTION
**Pull request by edgarriba**
goes with https://github.com/farm-ng/amiga-dev-kit/pull/202

image

### CODE REVIEW

File: py/examples/fastapi_backend/main.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    return StreamingResponse(generate_data(), media_type="text/event-stream")
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+if __name__ == "__main__":
+    import uvicorn
```
Comment (Hackerman342) [inline py/examples/fastapi_backend/main.py:88]: Import at top

File: py/examples/fastapi_backend/main.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+    for config in config_list.configs:
+        # create the event client
+        client = EventClient(config=config)
+
+        # add the client to the clients dict
+        clients[config.name] = client
+
+    # TODO (tony-ologic): review later so that we can access the website over tailscale
+    # without port forwarding.
```
Comment (Hackerman342) [inline py/examples/fastapi_backend/main.py:108]:
```suggestion
```

**Review by Hackerman342 — COMMENTED** [review]
@guilhermedemouraa to test

**Comment by edgarriba** [comment]
@Hackerman342 @guilhermedemouraa i finally merged the backend/frontend into the same example so it's more a full example of how to make a web app leveraging the backend microservices. So, please review again

**Review by guilhermedemouraa — APPROVED** [review]
Tested :ok_hand:

**Pull request merged**